CHANGELOG.md

```diff
@@ -57,7 +70,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Initial release ([#1](https://github.com/MetaMask/action-npm-publish/pull/1))
 
-[Unreleased]: https://github.com/MetaMask/action-npm-publish/compare/v3.2.0...HEAD
+[Unreleased]: https://github.com/MetaMask/action-npm-publish/compare/v3.2.1...HEAD
+[3.2.1]: https://github.com/MetaMask/action-npm-publish/compare/v3.2.0...v3.2.1
```

Comment:
The links were invalid:
```suggestion
[Unreleased]: https://github.com/MetaMask/action-npm-publish/compare/v4.0.0...HEAD
[4.0.0]: https://github.com/MetaMask/action-npm-publish/compare/v3.2.0...v4.0.0
```

The changelog linter isn't running in this repo I guess.

Reply:
no that was me! i screwed up the undo